commands/upload/upload.go

```diff
@@ -73,7 +73,7 @@ func Upload(ctx context.Context, req *rpc.UploadReq, outStream io.Writer, errStr
 }
 
 func runProgramAction(pm *packagemanager.PackageManager,
-	sketch *sketches.Sketch, importDir string, fqbnIn string, port string,
+	sketch *sketches.Sketch, importFile string, importDir string, fqbnIn string, port string,
```

Comment:
We can avoid repeating `string` for each argument here.

Reply:
:+1: 
I've removed also the repetition in `outStream, errStream`